### ISSUE TITLE
Change alt text for Downtown LA Home Unite Us location image on Events page

### ISSUE THREAD

**Issue by raswani2023** [hackforla/website#3315]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will change the alt text for the Downtown LA Home Unite Us location on the [Events page](https://www.hackforla.org/events/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [ ] In the file `_includes/events-page/our-locations-content.html`, change [line 121](https://github.com/hackforla/website/blob/gh-pages/_includes/events-page/our-locations-content.html#L121) of the alt text:

from
```
<img id="mini-img" src="../assets/images/projects/home-unite-us.png" alt="Home Unite Us Project" />
```

to
```
<img id="mini-img" src="../assets/images/projects/home-unite-us.png" alt="" />
```

- [ ] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic https://github.com/hackforla/website/issues/3132 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue https://github.com/hackforla/website/issues/3132 to the New Issue Approval column and remove the Dependency label

#### Resources/Instructions
This issue is part of epic https://github.com/hackforla/website/issues/3132
[Events page](https://www.hackforla.org/events/)
[Alt text decision Tree](https://www.w3.org/WAI/tutorials/images/decision-tree/)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
Directory to find the page once you have it in your IDE: `_includes/events-page/our-locations-content.html`
Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/f4fab7442666d5c6a8910f4f8d059c2bc43e373c/_includes/events-page/our-locations-content.html#L121

**Comment by tamara-snyder** [comment]
Hi @raswani2023, great work creating this issue. Please remember to add any new issues you create to the Project Board. Thanks!

**Comment by JessicaLucindaCheng** [comment]
- Due to changes in #3027, this issue is no longer valid because that image no longer exists on the page.
